src/harness/testcases/testdata/device_c.json

```diff
@@ -0,0 +1,20 @@
+{
+  "fccId": "test_fcc_id_a2",
```

Comment:
Is there a reason you're not using "test_fcc_id_c"? If you're using this to reference the category, maybe we should change device_{a,b}.json to match ("test_fcc_id_a1")?